src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -114,6 +116,7 @@ const SidebarPanels = ({
                             onVersionHistoryClick={onVersionHistoryClick}
                             startMarkName={MARK_NAME_JS_LOADING_ACTIVITY}
                             {...activitySidebarProps}
+                            refreshIdentity={refreshIdentity}
```

Comment:
Should these go after the `...activitySidebarProps` for precedence (overwritten by a parent)

Reply:
I can see what you're saying but I think `refreshIdentity` is a reflection of `ContentSidebar` internal state rather than a prop passed in from the top level user of the element so I'd rather not let them override it